float64[] position # xyzYPR: position +eluer angle

